com.unity.netcode.gameobjects/Tests/Runtime/DistributedAuthority/DistributedAuthorityCodecTests.cs

```diff
@@ -30,7 +30,7 @@ internal class DistributedAuthorityCodecTests : NetcodeIntegrationTest
         private NetworkManager Client => m_ClientNetworkManagers[0];
 
         private string m_TransportHost = Environment.GetEnvironmentVariable("NGO_HOST") ?? "127.0.0.1";
-        private const int k_TransportPort = 7777;
+        private const int k_TransportPort = 7788;
```

Comment:
Can we move to a environment variable configurable port? Along the lines of `Environment.GetEnvironmentVariable("NGO_PORT")`

Reply:
Yeah, I realised hard-coding the port would be an issue for the cmb runtime when they test against older versions. Updated so both sides are configurable and the default is still `7777`